src/chatgpt-log.ts

```diff
@@ -46,49 +19,61 @@ export class ChatGptLog {
       topP: request?.data?.top_p,
       functionCalls: request?.data?.functions,
       maxTokens: request?.data?.max_tokens,
-      startTime: prompt?.timestamp,
-      sentTime: request?.timestamp,
+      startTime: request?.data.startTime,
```

Comment:
I don't think that startTime and sentTime are in the request object. These are meta fields.

Reply:
I checked now, Start time is in the interface for Request, do want me to remove it?